lib/ui/painting/image.cc

```diff
@@ -28,7 +29,10 @@ void CanvasImage::RegisterNatives(tonic::DartLibraryNatives* natives) {
 
 CanvasImage::CanvasImage() {}
 
-CanvasImage::~CanvasImage() {}
+CanvasImage::~CanvasImage() {
+  SkImage* image = image_.release();
+  Threads::IO()->PostTask([image]() { image->unref(); });
```

Comment:
I wanted to ensure that there is no race between the unref when image_ is destructed (on the UI thread) and the unref when the captured copy is destructed (on the IO thread).

With the release() there is only one unref